api.go

```diff
@@ -867,6 +912,34 @@ func (lapi *Client) UpdateTaskStatus(id string, phase string, progress float64)
 	return nil
 }
 
+func (lapi *Client) CreateTranscodeTask(assetId string, name string, profile Profile) (*CreateTranscodeTaskResp, error) {
+	var (
+		url    = fmt.Sprintf("%s/api/asset/transcode", lapi.chosenServer)
+		input  = &createTranscodeTaskRequest{AssetId: assetId, Name: name, Profile: profile}
+		output CreateTranscodeTaskResp
+	)
+	err := lapi.doRequest("POST", url, "transcode_task", "", input, &output)
+	if err != nil {
+		return nil, err
+	}
+	glog.V(logs.DEBUG).Infof("Created transcode task id=%s assetId=%s status=%s type=%s", output.Task.ID, output.Asset.ID, output.Task.Status.Phase, output.Task.Type)
+	return &output, nil
```

Comment:
Can consider returning just the `output.Asset, output.Task, nil` here as well, but this one I'm not so sure.